### PR TITLE
fix bigquery push

### PR DESCRIPTION
**Pull request by nicflores**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Precog Push
+
+This projject contains various scripts for pushing to various datasources. Ideally these scripts will eventually become connectors.
```
Comment (alissapajer) [inline README.md]: s/projject/project/

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+This script makes use of the `gcloud` command:
+
+```
+gcloud auth application-default print-access-token
+``` 
+
+which in order to execute successfully and get a useful token you'll need gcloud account.
+
+To sart, issue the command
```
Comment (alissapajer) [inline README.md]: s/sart/start/

**Comment by alissapajer** [comment]
@nicflores found a couple more minor typos, I'll submit a patch since that's as easy as commenting them on here

**Comment by nicflores** [comment]
thanks for taking a look @alissapajer!

**Comment by beckyconning** [comment]
awesome thanks!

**Pull request merged**